### DISCUSSION TITLE
Installation

### DISCUSSION THREAD

**Discussion by mluqmanbukhori** [RobyRew/Lenovo-Ideapad-Flex-5-14IIL05_Hackintosh_OpenCore#7] [Q&A]
when is it released?
I have a lenovo ideapad flex 5 laptop, exactly the same

**Comment by mluqmanbukhori** [comment]
Okey, thanks full

**Comment by bhaskaranjana** [comment]
Hello Roby,
I recently got a flex 5 14IIL05.

I have experience with hackintoshing in past.

I tried everything I could (didn't sleep for two days) but I can't get the Mac os Installer to boot. Upon selecting the (external) dmg option in open core boot menu, I get an error known as [EB|#LOG:EXITBS:START].
I tried all the troubleshooting from dortania's guide[](https://dortania.github.io/OpenCore-Install-Guide/troubleshooting/extended/kernel-issues.html#stuck-on-eb-log-exitbs-start)  but I'm stuck on this.
![PXL_20211106_024126617](https://user-images.githubusercontent.com/32594474/140598303-fa7d0fe5-d26f-4261-8e46-6f8a6fc5f1dc.jpg)

Can you tell me how are you progressing? 

If you could, please add me on instagram so that we could discuss with ease. My username is _thekeep3r_

My specs
proc - 5 1035G1
Ssd - Samsung
Intel wifi 6

Should be the same model

**Comment by RobyRew** [comment]
@bhaskaranjana and @mluqmanbukhori I just released my first EFI for this laptop, I've not testet yet, but If you want to give it a try, now you can.

**Comment by vicrenlopez14** [comment]
What an amazing job as I see 👏, congrats!!. Still a shame for me having the Ryzen's version 😅.

**Comment by RobyRew** [comment]
> What an amazing job as I see 👏, congrats!!. Still a shame for me having the Ryzen's version 😅. 

Wow, that's very hard to hackintosh, APU are very complicated in terms of hackintosh...
Sorry man. :(

**Comment by EheDommy** [comment]
First of all, the “Install MacOS” option isn’t there when get to the opencore bootloader. After countless hours of reasearch, I found the problem in the config.plist in the drivers section. Once I fixed that and got the option to Install MacOS, it got an error that is [EB|#LOG:EXITBS:START] please fix the EFI. I got the same laptop but I got an i3 1005g1.

**Comment by RobyRew** [comment]
> First of all, the “Install MacOS” option isn’t there when get to the opencore bootloader. After countless hours of reasearch, I found the problem in the config.plist in the drivers section. Once I fixed that and got the option to Install MacOS, it got an error that is [EB|#LOG:EXITBS:START] please fix the EFI. I got the same laptop but I got an i3 1005g1.

Thanks man,I have not had much time to look at any of this, I have been doing other things these holidays, I hope to reactivate soon to update my repositories, and this is my sister's laptop, I also do not want to do tests that may affect it negatively, if you can not do it function activate the "Watch" and it will notify you when I put a release.

Or if you find something you can make a PR

**Comment by EheDommy** [comment]
Any updates? I'm still getting [EB|#LOG:EXITBS:START].

**Comment by RobyRew** [comment]
> Any updates? I'm still getting [EB|#LOG:EXITBS:START].

Hey @EheDommy you can try this current #6  for now, Big Sur with a lot of things to be fixed...
